resources/puppetlabs/lein-ezbake/template/foss/ext/debian/ezbake.init.erb

```diff
@@ -118,27 +109,24 @@ get_status_q()
 do_restart()
 {
     do_stop
-    case "$?" in
-        0|1)
```

Comment:
hmm, so, today, if we get an exit code of 1 on the stop, we just ignore it and try to start anyway?